public/components/integrations/components/setup_integration.tsx

```diff
@@ -185,6 +188,7 @@ export function SetupIntegrationForm({
     [] as Array<{ label: string }>
   );
   const [isSuggestionsLoading, setIsSuggestionsLoading] = useState(true);
+  const [isBlurred, setIsBlurred] = useState(false);
```

Comment:
there is also onBlur function that comes along with most elements that you can use and avoid this separate re-implementation

Reply:
Where is it documented? I'd heard about it but didn't find any references.